CHANGELOG.md

```diff
@@ -1,3 +1,66 @@
+# [8.6.0-next.7](https://github.com/readmeio/rdme/compare/v8.6.0-next.6...v8.6.0-next.7) (2023-02-13)
```

Comment:
there are a few formatting discrepancies in this file now but I really care to be honest